### DISCUSSION TITLE
surveymarks/tt-19-t/

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#26] [Announcements]
# surveymarks/tt-19-t/

http://localhost:4000/surveymarks/tt-19-t/

**Comment by jengalas** [comment]
Posted by Sean
March 22, 2023 at 12:10 pm
_Imported from WordPress_

Where was the Spangenberg School? Any ideas? Where’s the original description?

**Reply by jengalas** [reply]
Posted by Zhanna
March 26, 2023 at 9:25 pm
_Imported from WordPress_

The only description I have is the one above, listed as “Official USGS Description.” I haven’t found much online about the Spangenberg School, but if I can track down a copy of “The Rural Schools of Wayne County” by Mimi Steffen, that may help! I’ll try to find it.
